README.md

```diff
@@ -87,6 +92,32 @@ make stop
 make clean
 ```
 
+## Deploying
+
+All deployments are run by the [GDS Concourse][gds-concourse].
+
+The process amounts to:
+
+1. Test + Lint
+2. Build a deployment image
+3. Push deployment image to the GovWifi ECR
+4. Run migrations using the deployment image
+5. restart the ECS services using the new deployment image
+
+### Staging
+
+Deployments to Staging are run automatically whenever a change to the `master` branch is made.
+
+### Production
+
+Deployments to Production must be manually triggered.
+You can do this by going to the [production deployment pipeline][prod-deploy-pipeline], and starting the "Confirm Deploy to Production" task.
```

Comment:
Can you add a note on how to start this?
"Click on the plus button in the top right corner"